semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java

```diff
@@ -578,30 +584,6 @@ public final class SemanticAttributes {
    */
   public static final AttributeKey<Long> CODE_COLUMN = longKey("code.column");
 
-  /** HTTP request method. */
-  public static final AttributeKey<String> HTTP_METHOD = stringKey("http.method");
-
-  /** <a href="https://tools.ietf.org/html/rfc7231#section-6">HTTP response status code</a>. */
-  public static final AttributeKey<Long> HTTP_STATUS_CODE = longKey("http.status_code");
-
-  /**
-   * Kind of HTTP protocol used.
-   *
-   * <p>Notes:
-   *
-   * <ul>
-   *   <li>If {@code net.transport} is not specified, it can be assumed to be {@code IP.TCP} except
-   *       if {@code http.flavor} is {@code QUIC}, in which case {@code IP.UDP} is assumed.
-   * </ul>
-   */
-  public static final AttributeKey<String> HTTP_FLAVOR = stringKey("http.flavor");
-
-  /**
-   * Value of the <a href="https://www.rfc-editor.org/rfc/rfc9110.html#field.user-agent">HTTP
-   * User-Agent</a> header sent by the client.
-   */
-  public static final AttributeKey<String> HTTP_USER_AGENT = stringKey("http.user_agent");
```

Comment:
Can you add a deprecated attribute to https://github.com/open-telemetry/opentelemetry-java/blob/main/buildscripts/semantic-convention/templates/SemanticAttributes.java.j2#L111 to minimize breaking changes?

Reply:
This code is auto-generated from the spec changes and the tool does not generate removed attributes. 
I can add them manually, but they will need some handholding next time. WDYT?

Reply:
@lmolkova There are provisions for this in the template itself.

See the code after this magic line:

```
  {%- if class == "SemanticAttributes" %}
```

For example:

* https://github.com/open-telemetry/opentelemetry-java/blob/38bf4d8ec0d794f9be77d2fe33b1ed99f5262eef/buildscripts/semantic-convention/templates/SemanticAttributes.java.j2#L121-L128

Reply:
@marcalff thank you for the pointer!!